lntest/itest/lnd_multi-hop_htlc_local_timeout_test.go

```diff
@@ -221,10 +221,10 @@ func testMultiHopHtlcLocalTimeout(net *lntest.NetworkHarness, t *harnessTest,
 		t.Fatalf("bob should have pending htlc but doesn't")
 	}
 
-	// Now we'll mine an additional block, which should confirm Bob's commit
-	// sweep. This block should also prompt Bob to broadcast their second
-	// layer sweep due to the CSV on the HTLC timeout output.
-	slowMineBlocks(t, net, 1, 1)
+	// Now we'll mine two additional blocks, the first should confirm Bob's
+	// commit sweep. The following block should prompt Bob to broadcast
+	// their second layer sweep due to the CSV on the HTLC timeout output.
+	slowMineBlocks(t, net, 2, 1)
```

Comment:
How did the blocks being mined very quickly mask this?

Reply:
Due to the interaction with the previous `mineBlocks`, when fast mining, the sweeper (which uses a 2s timeout in itests) is not reliably triggered every single block.

So when you fast mine, you actually pass the first eligible block where some sweeping tx would be broadcast. `slowMineBlocks` ensures the sweeper is triggered at every block.